lib/auth/recordingencryption/recordingencryptionv1/service_test.go

```diff
@@ -300,3 +317,100 @@ func (f *fakeKeyRotater) RollbackRotation(ctx context.Context) error {
 func (f *fakeKeyRotater) GetRotationState(ctx context.Context) ([]*recordingencryptionv1pb.FingerprintWithState, error) {
 	return f.keys, nil
 }
+
+type fakeSessionStreamer struct{}
+
+func (f *fakeSessionStreamer) StreamSessionEvents(ctx context.Context, sessionID session.ID, startIndex int64) (chan apievents.AuditEvent, chan error) {
+	returnChan := make(chan apievents.AuditEvent, 1)
+	errChan := make(chan error, 1)
+	close(errChan)
+	events := eventstest.GenerateTestSession(eventstest.SessionParams{
+		UserName:  "alice",
+		SessionID: string(sessionID),
+		ServerID:  "testcluster",
+		PrintData: []string{"net", "stat"},
+	})
+	returnChan <- events[len(events)-1]
```

Comment:
Should we stream at least one more event than the last to make sure they're they're ignored?

Reply:
This is already tested in sessionend_test.go.